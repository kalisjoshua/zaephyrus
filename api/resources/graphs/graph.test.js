const {createGraph} = require("./graph.js")

describe("createGraph", () => {
  test("create instance", () => {
    const inst = createGraph('b=main&c=&t=v1.0.0&t=latest&t=release-candidate&t=stable&b=another&c=testing something out some really long commit first line subject not advisable to do this in real life&b=main&c=hotfix&c=new feature')

    expect(inst).toMatchSnapshot()
  })
})
