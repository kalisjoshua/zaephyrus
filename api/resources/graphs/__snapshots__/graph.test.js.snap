// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createGraph create instance 1`] = `
"
    <svg  style=\\"height: 200px; width: 960px;\\" viewBox=\\"0 0 960 200\\" xmlns=\\"http://www.w3.org/2000/svg\\">
      <defs>
        <style type=\\"text/css\\"><![CDATA[
          svg {}
        ]]></style>
      </defs>

      <g><line stroke=\\"#603AA1\\" stroke-width=\\"8\\" x1=\\"40\\" x2=\\"40\\" y1=\\"80\\" y2=\\"40\\"></line>
<line stroke=\\"#603AA1\\" stroke-width=\\"8\\" x1=\\"40\\" x2=\\"40\\" y1=\\"160\\" y2=\\"80\\"></line>
<line stroke=\\"#1C6EF2\\" stroke-width=\\"8\\" x1=\\"40\\" x2=\\"80\\" y1=\\"160\\" y2=\\"120\\"></line>
<circle cx=\\"80\\" cy=\\"120\\" fill=\\"#1C6EF2\\" r=\\"10\\"></circle>
<circle cx=\\"40\\" cy=\\"80\\" fill=\\"#603AA1\\" r=\\"10\\"></circle>
<circle cx=\\"40\\" cy=\\"40\\" fill=\\"#603AA1\\" r=\\"10\\"></circle>
<circle cx=\\"40\\" cy=\\"160\\" fill=\\"#603AA1\\" r=\\"10\\"></circle></g>
    </svg>"
`;
